doc/en/components/charts/chart-types-bar.md

```diff
@@ -6,23 +6,118 @@ mentionedTypes: ["XamDataChart"]
+$PlatformShort$ Bar Charts, Bar Graphs, or Horizontal Bar Charts, are among the most common category chart types used to quickly compare frequency, count, total, or average of data in different categories with data encoded by horizontal bars or equal width and differing lengths. They are ideal for showing variations in the value of an item over time. Data is represented using a collection of rectangles that extend from the left to right of the chart towards the values of data points. Like a Column Chart, or Column Graph, where bars are displayed vertically (up and down), the bar chart is rendered like the column chart, but with 90 degrees clockwise rotation and a horizontal display (left to right).
 
-## $PlatformShort$ Simple Bar Chart
+There are several common use cases for choosing a Bar Chart: 
 
-<!-- TODO copy and combine content (code snippets, description) from these topics:
-    data-chart-type-category-bar-series.md
--->
+- You need to show trends over time or a numeric value change in a category of data 
+- You need to compare data values of 1 or more data series 
+- You want to show a part-to-whole comparison 
+- You want to show top or bottom percentage of categories 
+- Analyzing multiple data points grouped in sub-categories (Stacked Bar) 
+
+The $PlatformShort$ Bar Chart includes several variants based on your data or how you want to tell the correct story with your data. These include: 
+
+- Grouped Bar Chart 
+- Stacked Bar Chart 
+- Polar Bar Chart 
+- Stacked 100 Bar Chart 
+
+Bar Chart Best Practices: 
+
+- Start you numeric Axis at 0 
+- Use a single color for the bars  
+- Be sure the space separating each bar is 1/2 the width of the bar itself 
+- Be sure ranking or comparing ordered categories (items) are  sorted in increasing or decreasing order 
+- Right-align category values on the Y-Axis (left side labels of chart) for readability 
+
+Don't Use a Bar Chart When: 
+
+- You have too much data so the Y-Axis can't fit in the space or is not legible 
+- You need a Detailed Time-Series analysis  - consider a Line Chart with a Time-Series for this type of data.
+
+Bar Chart Data Structure: 
+
+- The data source must be an array or a list of data items 
+- The data source must contain at least one data item  
+- The list must contain at least one data column (string or date time)  
+- The list must contain at least one numeric data column 
+
+Common $PlatformShort$ Bar Chart Scenarios: 
+
+- Sales Management 
+- Inventory Management 
+- Stock Charts 
+- Any String Value Comparing a Numeric Value or Time-Series Value 
+
+## $PlatformShort$ Bar Chart Example
+
+In this example, the bar chart is comparing the total number of Olympic Medals (the Category) won between the USA and Russia from 1996 to 2016. The Y-Axis, or labels on the left of the chart, are displaying the time-series values of Year. The X-Axis, or labels on the bottom of the chart, are displaying the number of medals won.
```

Comment:
Is this going to be the example, or are we going to use the new scenario?

Reply:
This is what Jason drafted up but if you have a better scenario I think we can go ahead and edit the paragraph accordingly.

Reply:
My guess is he's just writing with the current examples. Because what I had to do was to come up with new ones. The question is did he review them or not (the Abstract link I sent)?

Reply:
@tosharky You're right. Jason is writing an overview of the existing samples. He also mentioned in a few places that we need new samples. We can ask him to rewrite/edit the overview when the new samples are in place

Reply:
https://share.goabstract.com/1e0cf345-060c-49d3-8e04-6fb02aa21e2d?collectionLayerId=c2eae4b8-1622-41e2-b706-867a4fd3a10f&mode=design

Reply:
And this is the data for all the examples:
[Stacked Bar Chart.xlsx](https://github.com/IgniteUI/igniteui-xplat-docs/files/5679972/Stacked.Bar.Chart.xlsx)